gap/AlmostSimpleDefiningCharacteristic.gi

```diff
@@ -0,0 +1,766 @@
+# TODO: In some functions we choose particular elements of the generating set.
+#       We have to check whether the group generated by these elements is equal to the group
+#       generated in magma (Or have some other kind of check whether the output is correct).
+#       Write functions which output the generators for the functions below?
+#       Talk with Sergio about this
+
+
+
+####################################################################################
+############################# OverSmallerField #####################################
+####################################################################################
+
+
+# The following functions are from the Recog Package.
+# Since some functions call other functions from the Recog Package, one
+# needs to load the Recog Package.
+
+
+# From the Recog Package but slightly modified (by DR).
```

Comment:
Did you copy, paste that? Could we instead extend the function in recog such that you can also use it here?